app/policies/user_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class UserPolicy < ApplicationPolicy
```

Comment:
How about doing something similar for rents? so we authorize rents and only show the rents relevant to the logged in user